api/serializers/course.py

```diff
@@ -1,7 +1,8 @@
 from rest_framework import serializers
 
 from accounts.models import MyAnonymousUser
-from api.serializers import QuestionSerializer, UQJSerializer
+from api.serializers.question import QuestionSerializer
```

Comment:
Either fix all of them or leave them as they are. also the same for __init__.py s. I would recommend to fix them all if you want :D

Reply:
fixing as in making it so every import come from the specific file it references?

Reply:
No, its importing all the serializers in __init__.py and having one line of import here.